legacy/src/api/assets/assets.md

```diff
@@ -106,24 +106,26 @@ Gift cards have the following fields along with the base asset fields.
 <a name="tokens">
 ### Tokens (EXPERIMENTAL)
 
-Tokens are assets which can be spent only once. They are usually tied to a
-small set of merchants and have an expiry date. Token value may be set in
-multiple currencies and is the same for all tokens of the same type.
+Tokens are assets which can only be spent in full. 
+
+Every token is associated with a collection, which defines the branding and general rules for the tokens, such as active duration.
 
 Tokens have the following fields along with the base asset fields.
 
 {% h4 Fields %}
 
-|     Field     |             Type             |                                         Description                                          |
-| :------------ | :--------------------------- | :------------------------------------------------------------------------------------------- |
-| value         | Array {% opt %}              | The [Monetary Amounts][] representing the token's nominal value in its supported currencies. |
-| activeFrom    | {% dt Timestamp %} {% opt %} | The date when the asset becomes spendable.                                                   |
-| expiresAt     | {% dt Timestamp %} {% opt %} | The date when the asset expires.                                                             |
-| img           | String {% opt %}             | The img URL of the token.                                                                    |
-| issuer        | String {% opt %}             | The identifier for the issuer of the token.                                                  |
-| issuerWebsite | String {% opt %}             | The URL of the issuer of the token.                                                          |
-| issuerImg     | String {% opt %}             | The img URL of the issuer that the token belongs to.                                         |
-
+|     Field      |             Type             |                                                 Description                                                 |
+| :------------- | :--------------------------- | :---------------------------------------------------------------------------------------------------------- |
+| collectionId   | String                       | The [token collection](#token-collection) that will govern the branding and redemption rules for the token. |
+| idempotencyKey | String                       | Client-supplied identifier that prevents double creation.                                                   |
```

Comment:
FYI this is not a field that a token asset has. It is only supplied in the create API call.

Reply:
Yep, have removed idempotencyKey field. Thanks for pointing it out 💯 🙌